### PR TITLE
Publish t2iapi to pypi

### PR DESCRIPTION
**Pull request by leon1995**
In order to easily distribute the t2iapi pyhon package built from the protobuf sources, upload t2iapi python package to pypi.org

### CODE REVIEW

File: .github/workflows/build-python.yml [hunk 1]
```diff
@@ -47,19 +48,19 @@ jobs:
       - uses: actions/checkout@v2
```
Comment (antipovi) [inline .github/workflows/build-python.yml:48]: shouldn't this be v3?

File: .github/workflows/build-python.yml [hunk 2]
```diff
@@ -72,3 +73,17 @@ jobs:
           name: html_unit_test
           path: unittest_results/*.html
           retention-days: 1
+  deploy-github-packages:
```
Comment (antipovi) [inline .github/workflows/build-python.yml:76]: i would suggest to ranme this to something pypi specific
"publish to pypi"?

File: python/setup.py [hunk 3]
```diff
@@ -4,39 +4,42 @@
...
-    Otherwise, return '0.0.0'.
+    Get the package and development release version from the respective environment parameter, if available.
+    Otherwise, return '0.0.0.dev0'.
+    Following version scheme from https://peps.python.org/pep-0440/#version-scheme
 
     @return: package version
     """
-    return os.getenv('build_version', default='0.0.0')
+    package = os.getenv('BASE_PACKAGE_VERSION', default='0.0.0')
+    dev_release = f'dev{os.getenv("GITHUB_RUN_NUMBER", default=0)}'
```
Comment (antipovi) [inline python/setup.py:31]: rename the variable, it implies the release version
maybe "dev_run_number" or simply "run_number"?

Comment (ldeichmann) [reply python/setup.py:31]: https://peps.python.org/pep-0440/#developmental-releases would suggest `dev_release_segment` or similiar ;)

File: .github/workflows/build-python.yml (hunk 1 above)
Comment (leon1995) [reply .github/workflows/build-python.yml:48]: changed to `v3`

File: .github/workflows/build-python.yml (hunk 2 above)
Comment (leon1995) [reply .github/workflows/build-python.yml:76]: renamed to `deploy-pypi-packages`

File: python/setup.py (hunk 3 above)
Comment (leon1995) [reply python/setup.py:31]: renamed to `dev_release_segment`

**Review by antipovi — APPROVED** [review]

**Pull request merged**